source/delegate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/delegate",
-  "version": "1.5.8",
+  "version": "2.6.8-beta.1",
```

Comment:
I believe it should be beta.0 before beta.1 - just as we would have 2.6.0 before 2.6.1